src/mgr/DaemonServer.h

```diff
@@ -251,6 +253,9 @@ class DaemonServer : public Dispatcher, public md_config_obs_t
 
   void update_task_status(DaemonKey key,
 			  const std::map<std::string,std::string>& task_status);
+private:
+  // -- op tracking --
+  OpTracker op_tracker;
```

Comment:
Oops, why do have two instances of `OpTracker op_tracker`? One is here (`DaemonServer`), the other one in `Mgr`.

Reply:
Intention was to track ops between mon and mgr like osdmap, monmap updates from mon. I have removed the `op_tracker` from Mgr.